frontend/src/scenes/insights/insightDataLogic.tsx

```diff
@@ -187,21 +193,62 @@ export const insightDataLogic = kea<insightDataLogicType>([
                 actions.setInsightData({ ...values.insightData, result })
             }
         },
-        loadInsightSuccess: ({ insight }) => {
```

Comment:
bit confused why this was here in the first place 😅 , maybe @thmsobrmlr has more context.
Though wanted to check if you had any reasons for removing it? If it's just not needed can also remove it from line 58
```
insightLogic,
['setInsight', 'loadInsightSuccess'],
```